MCprep_addon/util.py

```diff
@@ -186,19 +186,19 @@ def obj_copy(base, context=None, vertex_groups=True, modifiers=True):
         
 
 @functools.cache
-def getBV():
+def getBV(major_version, minor_version):
```

Comment:
Maybe we rename to: `def min_bv(version)` which better describes which it does - and then version is just a tuple input (so we can be more granular).